src/meta/buildTags.tsx

```diff
@@ -119,13 +119,13 @@ const buildTags = (config: BuildTagsParams) => {
   }
 
   const noindex =
-    config.noindex ||
-    defaults.noindex ||
-    config.dangerouslySetAllPagesToNoIndex;
+    config.noindex != null
```

Comment:
`noindex` should be a _boolean_, otherwise _undefined_.

Reply:
@nyedidikeke Sorry, I don't get it. With this code, `noindex` of each page is respected if it's given, otherwise, global `dangerouslySetAllPagesToNoIndex` is reffered.